### PR TITLE
[v6r14] Propagating hideExceptions flag when creating StorageElements

### PR DESCRIPTION
**Pull request by fstagni**
In this PR I also:
- Removed SRMSpaceTokenDeployment Accounting type
- fixed an important bug in the pilot, which is a consequence of latest fixes in dirac-wms-cpu-normalization script
- set required GSI version to 0.6.3
- Added support for all protocols for SEs managed by RSS
- Added ARC_PLUGIN_PATH to newly created bashrc and cshrc files
- Re-try few times to update the RSS SE cache before giving up
- fix typo in get-proxy-info: actime -> actimeleft

### CODE REVIEW

**Comment by fstagni** [comment]
This also means that v6r14p1 has a buggy pilot and should be probably removed

**Comment by chaen** [comment]
As a general remark, since this hideExceptions is a named parameter, please name it when calling it. 
Otherwise, OK

**Comment by fstagni** [comment]
Yes, done

File: WorkloadManagementSystem/PilotAgent/pilotCommands.py [hunk 1]
```diff
@@ -650,7 +650,7 @@ def execute( self ):
       self.exitWithError( retCode )
 
     # HS06 benchmark
-    cpuNormalizationFactor = float( cpuNormalizationFactorOutput.replace( "Normalization for current CPU is ", '' ).replace( " HS06", '' ) )
+    cpuNormalizationFactor = float( cpuNormalizationFactorOutput.replace( "Estimated CPU power is ", '' ).replace( " HS06", '' ) )
```
Comment (phicharp) [inline WorkloadManagementSystem/PilotAgent/pilotCommands.py]: My apologies for this... I had not realized we were calling the script and parsing the output! Why not call the same method called by the script?
Take care also that if MJF is implemented the script returns two lines: one with that string and one with the MJF value...

**Comment by phicharp** [comment]
Review OK

**Comment by phicharp** [comment]
Final review OK

File: Core/Security/VOMS.py [hunk 2]
```diff
@@ -81,6 +87,24 @@ def getVOMSProxyFQAN( self, proxy ):
     return self.getVOMSProxyInfo( proxy, "fqan" )
 
   def getVOMSProxyInfo( self, proxy, option = False ):
+    """ Returns information about a proxy certificate (both grid and voms).
+        Available information is:
+          1. Full (grid)voms-proxy-info output
+          2. Proxy Certificate Timeleft in seconds (the output is an int)
+          3. DN
```
Comment (atsareg) [inline Core/Security/VOMS.py]: What should be the convention for the arguments description ? This one or 
:param string param_name: parameter description

Comment (chaen) [inline Core/Security/VOMS.py]: I like the real doctstring, :param....

Comment (atsareg) [inline Core/Security/VOMS.py]: I think this is what sphinx likes too

**Pull request merged**